libs/shared/left-side-menu/src/lib/shared-left-side-menu.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from "react";
+
+import './shared-left-side-menu.scss';
+
+import {
+  Drawer,
+  List,
+  ListItem,
+  Typography,
+  ListItemText,
+  IconButton
+} from '@material-ui/core';
+
+import { 
+  ChevronLeft as ChevronLeftIcon,
+  Menu as MenuIcon
+ } from '@material-ui/icons';
+
+interface Type {
+  name: string;
+  path: string
+}
+
+export interface SharedLeftSideMenuProps {
+  title: string,
+  menuItemsArray: Array<Type>
+}
+
+export const SharedLeftSideMenu = (props: SharedLeftSideMenuProps) => {
+  const [state, setState] = useState({
```

Comment:
rename state to isMenuOpen, setMenuOpen and on state set value not object useState(false)